IFC4x3/Properties/m/MitigatedRiskLikelihood_2mh3cZyunB0ONymM2LqXn/DocProperty.xml

```diff
@@ -1,5 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="MitigatedRiskLikelihood_2mh3cZyunB0ONymM2LqXn" Name="MitigatedRiskLikelihood" UniqueId="b0ac39a3-f38c-4b01-85fc-c16095d21c7f" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
Accidental deletion please revert, this element is still referenced in the Pset_Risk will cause an error in IfcDoc.

Reply:
This one also seems to have had a duplicate, the property preserved in the main branch has a $ at the end. Note within an xml href id, the $ character is invalid and suppressed within the file (but not necessarily the folder, perhaps we should review this?)  https://github.com/bSI-InfraRoom/IFC-Specification/tree/64f1fc0e8cdf8aba82d2b99a423ddefae2f5a2ba/IFC4x3/Properties/m

Reply:
It does need addressing, I do remember this $ i tried removing it but kept on getting put back by IFC doc if you've sorted it then good.